core/src/display_object/stage.rs

```diff
@@ -1034,7 +1050,7 @@ bitflags! {
     ///
     /// This is a bitflags instead of an enum to mimic Flash Player behavior.
     /// You can theoretically have both TOP and BOTTOM bits set, for example.
-    #[derive(Clone, Copy, Default, Collect)]
+    #[derive(Clone, Collect, Copy, Debug, Default, Eq, PartialEq)]
```

Comment:
nit: we usually put Collect on end, after stdlib derives. I know there are exceptions already, but this way it's slightly neater ;D

Reply:
Ah I didn't know about that, I'll move it back to the end then